uin_card_template.properties

```diff
@@ -0,0 +1 @@
dG9wLWJ1ZmZlciI+IDxsYWJlbCBjbGFzcz0ibmFtZS1oZWFkLWNvbG9yIj48Yj5OYW1lJm5ic3A7OiZuYnNwOzwvYj48L2xhYmVsPiA8c3BhbiBjbGFzcz0ibmFtZS1jb2xvciI+ICQhZnVsbE5hbWVfZW5nPC9zcGFuPiA8L2Rpdj4gPC90ZD4gPC90cj4gPHRyPiA8dGQgcm93c3Bhbj0iMSI+IDxkaXYgY2xhc3M9ImJsb2NrIHRvcC1idWZmZXIiPiA8bGFiZWwgY2xhc3M9Im5hbWUtaGVhZC1jb2xvciI+PGI+2KfYs9mFJm5ic3A7OiZuYnNwOzwvYj48L2xhYmVsPiA8c3Bhbj4kIWZ1bGxOYW1lX2FyYTwvc3Bhbj4gPC9kaXY+IDwvdGQ+IDwvdHI+IDx0cj4gPHRkIHJvd3NwYW49IjEiPiA8ZGl2IGNsYXNzPSJibG9jayB0b3AtYnVmZmVyIj4gPGxhYmVsIGNsYXNzPSJuYW1lLWhlYWQtY29sb3IiPjxiPkRPQi/Yqtin2LHZitiuJm5ic3A7OiZuYnNwOzwvYj48L2xhYmVsPiA8c3Bhbj4kIWRhdGVPZkJpcnRoPC9zcGFuPiA8L2Rpdj4gPC90ZD4gPC90cj4gPHRyPiA8dGQgY29sc3Bhbj0iMiI+IDxkaXYgY2xhc3M9ImJsb2NrIHRvcC1idWZmZXIiPiA8bGFiZWwgY2xhc3M9Im5hbWUtaGVhZC1jb2xvciI+PGI+R2VuZGVyJm5ic3A7OiZuYnNwOzwvYj48L2xhYmVsPiA8c3Bhbj4kIWdlbmRlcl9lbmc8L3NwYW4+ICZuYnNwOyZuYnNwOyA8bGFiZWwgY2xhc3M9Im5hbWUtaGVhZC1jb2xvciI+PGI+2KzZhtizJm5ic3A7OiZuYnNwOzwvYj48L2xhYmVsPiA8c3Bhbj4kIWdlbmRlcl9hcmE8L3NwYW4+IDwvZGl2PiA8L3RkPiA8L3RyPiA8dHI+IDx0ZCBjb2xzcGFuPSIzIj4gPGRpdiBjbGFzcz0icm93IiBzdHlsZT0ibWFyZ2luLXJpZ2h0OiAwcHg7IG1hcmdpbi1sZWZ0OiAtMTBweDsiPiA8ZGl2IGNsYXNzPSJjb2wtbWQtMiB0b3AtYnVmZmVyIj4gPGRpdiBjbGFzcz0iYmxvY2sgdG9wLWJ1ZmZlciI+IDx0YWJsZT4gPHRyPiA8dGQ+PGxhYmVsIGNsYXNzPSJuYW1lLWhlYWQtY29sb3IiPjxiPkFkZHJlc3MmbmJzcDs6Jm5ic3A7PC9iPjwvbGFiZWw+IDwvdGQ+IDx0ZD48c3Bhbj4kIWFkZHJlc3NMaW5lMV9lbmcsICQhYWRkcmVzc0xpbmUyX2VuZywgJCFhZGRyZXNzTGluZTNfZW5nLCAkIXJlZ2lvbl9lbmcsICQhcHJvdmluY2VfZW5nLCAkIWNpdHlfZW5nLCAkIXBvc3RhbENvZGUgPC9zcGFuPjwvdGQ+IDwvdHI+IDx0cj4gPHRkPjxsYWJlbCBjbGFzcz0ibmFtZS1oZWFkLWNvbG9yIj48Yj7YudmG2YjYp9mGJm5ic3A7OiZuYnNwOzwvYj48L2xhYmVsPiA8L3RkPiA8dGQ+PHNwYW4+JCFhZGRyZXNzTGluZTFfYXJhLCAkIWFkZHJlc3NMaW5lMl9hcmEsICQhYWRkcmVzc0xpbmUzX2FyYSwgJCFyZWdpb25fYXJhLCAkIXByb3ZpbmNlX2FyYSwgJCFjaXR5X2FyYSwgJCFwb3N0YWxDb2RlIDwvc3Bhbj4gPC90ZD4gPC90cj4gPC90YWJsZT4gPC9kaXY+IDwvZGl2PiA8L2Rpdj4gPC90ZD4gPC90cj4gPC90YWJsZT4gPGRpdj4mbmJzcDs8L2Rpdj4gPHRhYmxlIGNsYXNzPSJtYWluLXRhYmxlIiBzdHlsZT0iaGVpZ2h0OiAzMDBweCI+IDx0cj4gPHRkPiA8ZGl2IHN0eWxlPSJtYXJnaW4tbGVmdDogMTBweDsgbWFyZ2luLXJpZ2h0OiAxMHB4OyBib3JkZXI6IHNvbGlkIGJsYWNrIDFweDsgaGVpZ2h0OiAyNTBweDsgd2lkdGg6IDI1MHB4OyI+IDxkaXYgY2xhc3M9ImNvbC1tZC02Ij4gPGRpdiBjbGFzcz0iYmFyLWNvZGUtcGFkZGluZyI+PC9kaXY+IDxpbWcgc3JjPSQhUXJDb2RlIHN0eWxlPSJ3aWR0aDogMjUwcHg7IGhlaWdodDogMjUwcHg7IG1hcmdpbi10b3A6IC0yMHB4Ij4gPC9kaXY+IDwvZGl2PiA8L3RkPiA8dGQ+IDxkaXYgY2xhc3M9Im5hbWUtaGVhZC1jb2xvciBjb2wtbWQtNiI+IDxicj4gRm9yIGFueSBpc3N1ZXMgcGxlYXNlIGNvbnRhY3QgdXMgYXQ8YnI+IDxicj4gUmVnaXN0cmF0aW9uIFByb2NjZXNzb3IsSGFuZ2luZyBHYXJkZW5zLEdsb2JhbCBWaWxsYWdlIFRlY2ggUGFyaywgTXlzb3JlIFJkLFJWQ0UsQmVuZ2FsdXJ1LCBLYXJuYXRha2EgNTYwMDU5PGJyPiA8YnI+INmE2KPZitipINmF2LTYp9mD2YQg2YrYsdis2Ykg2KfZhNin2KrYtdin2YQg2KjZhtinINi52YTZiSA8YnI+2KfZhNiq2LPYrNmK2YQg2KfZhNmF2LnYp9mE2KzYjCDYp9mE2K3Yr9in2KbZgiDYp9mE2YXYudmE2YLYqSDYjCDYp9mE2YLYsdmK2Kkg2KfZhNi52KfZhNmF2YrYqSBUZWNoIFBhcmssIE15c29yZSBSZCxSVkNFIDU2MDA1OSDYqNmG2LrYp9mE2YjYsdmI2Iwg2YPYp9ix2YbYp9iq2KfZg9inIDxicj4gPGJyPiA8L2Rpdj4gPC90ZD4gPC90cj4gPC90YWJsZT4gPC9ib2R5PiA8L2h0bWw+
```

Comment:
As discussed keep the properties in a single template properties file storing multiple templates ask key value pairs in the same file.

Reply:
taken care